File: gita80/if/task3/main.dart
import 'dart:io';

void main(){
  double x,y;
  stdout.write("x: ");
  x = double.parse(stdin.readLineSync()??'');
  stdout.write("y: ");
  y = double.parse(stdin.readLineSync()??'');
  if(x==y) {
    print("$x : $y");
    return;
  }

  double half = (x+y)/2;
  double doubled = x*y*2;

  if(x>y) {
    [x,y]=[doubled,half];
  } else {
    [x,y]=[half,doubled];
  }
  print("$x : $y");
  

}

File: gita80/if/task3/main.cpp
#include <iostream>
using namespace std;

int main() {
    float x,y;
    cout<<"x: ";
    cin>>x;
    cout<<"y: ";
    cin>>y;
    if(x==y) {
        cout<<x<<" : "<<y;
        return 0;
    }

    float half = (x+y)/2;
    float doubled = x*y*2;

    float tempX = x;
    x = x<y?half:doubled;
    y = y<tempX?half:doubled; 


    cout<<x<<" : "<<y;
    

    return 0;
}
